### ISSUE TITLE
TextDecoder not found in nodejs env

### ISSUE THREAD

**Issue by connectdotz** [textileio/js-http-client#118]
calling `textile.contacts.search(name)` in jest intergration testing (nodejs based) and see the following error:
```
ReferenceError: TextDecoder is not defined

      at Object.start (../../node_modules/@textile/js-http-client/dist/helpers/handlers.js:72:29)
      at Call (../../node_modules/web-streams-polyfill/src/lib/helpers.ts:61:35)
      at InvokeOrNoop (../../node_modules/web-streams-polyfill/src/lib/helpers.ts:120:10)
      at startAlgorithm (../../node_modules/web-streams-polyfill/src/lib/readable-stream.ts:1543:12)
      at SetUpReadableStreamDefaultController (../../node_modules/web-streams-polyfill/src/lib/readable-stream.ts:1518:23)
      at SetUpReadableStreamDefaultControllerFromUnderlyingSource (../../node_modules/web-streams-polyfill/src/lib/readable-stream.ts:1553:3)
      at new ReadableStream (../../node_modules/web-streams-polyfill/src/lib/readable-stream.ts:119:7)
      at Object.<anonymous>.exports.streamHandler (../../node_modules/@textile/js-http-client/dist/helpers/handlers.js:68:12)
      at Contacts.<anonymous> (../../node_modules/@textile/js-http-client/dist/modules/contacts.js:89:31)
```

looks like this code won't run in nodejs, is that intended?

jest: 24.7.1
node: 10.15.1

**Comment by connectdotz** [comment]
looks like TextDecoder is added to global after v11: [Globals/TextDecoder](https://nodejs.org/dist/latest-v12.x/docs/api/globals.html#globals_textdecoder)... 

I think either to change the code to make it more compatible to older nodejs or to document the minimal nodejs version (>= 11.0)...

**Comment by carsonfarmer** [comment]
Closed with #119. Thanks @connectdotz!
